jenkins/test-history/gen_json.py

```diff
@@ -145,7 +145,10 @@ def _get_builds(self, job):
         except (ValueError, TypeError):
             pass
         else:
-            return (str(n) for n in xrange(latest_build, 0, -1))
+            # If it looks like a jenkins build number, return all the indices
+            if latest_build < 20160000000000:
```

Comment:
JSON is better-known than YAML, which makes it preferable for small/simple
files like this.

On Thu, Jun 2, 2016 at 10:46 AM, Jeff Grafton <notifications@github.com>
wrote:

> In jenkins/test-history/gen_json.py
> <https://github.com/kubernetes/test-infra/pull/81#discussion_r65585874>:
>
> > @@ -145,7 +145,10 @@ def _get_builds(self, job):
> >          except (ValueError, TypeError):
> >              pass
> >          else:
> > -            return (str(n) for n in xrange(latest_build, 0, -1))
> > +            # If it looks like a jenkins build number, return all the indices
> > +            if latest_build < 20160000000000:
>
> YAML seems a bit more friendly for humans than JSON, but I don't feel too
> strongly about that.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/test-infra/pull/81/files/44f4abdb8c0d6ae868e0b096dcf3a6097a8ab7e1#r65585874>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAM7nBk_Ok8wqLoElYZ9fyLkP75_UOv2ks5qHxcPgaJpZM4IoaZi>
> .
>